akka-stream/src/main/scala/akka/stream/impl/Modules.scala

```diff
@@ -46,13 +47,15 @@ private[akka] abstract class SourceModule[+Out, +Mat](val shape: SourceShape[Out
  */
 private[akka] final class SubscriberSource[Out](val attributes: Attributes, shape: SourceShape[Out]) extends SourceModule[Out, Subscriber[Out]](shape) {
 
+  override protected def label: String = "SubscriberSource"
```

Comment:
Is there a reason why `Logging.simpleName(this)` wouldn't be the canonical implementation hereof?